docs/rfc/rfc-1.adoc

```diff
@@ -41,9 +44,81 @@ and maintain the peg), we must turn to another token. Instead, the system pays
 operators in T tokens from a treasury funded by the T-token DAO and whenever it
 collects tBTC fees it uses those fees to buy T tokens back for its treasury.
 
+=== Minting And Unminting
+
+A user with an account balance can drain their account balance to mint TBTC.
+Similarly, a user can burn TBTC to refill their account balance.
+
+// FIXME: fill in this section with how much TBTC they can mint with regards to
+// their unswept balance
+
 == In Depth
 
-=== Minting
+[[group-size]]
+=== Group Size
+There are three main governable parameters that we need to choose:
+
+* The number of signers that can be controlled by the adversary before the key is in danger `threshold`
+* The number of group members required for a <<heartbeat,heartbeat>> to be considered "strong". `heartbeat`
+* The minimum number of members we're allowed to drop down to during the DKG
+  group formation re-try period. `dkg`
+* The total size of the group `groupSize`
+
+`threshold < heartbeat < dkg < groupSize`
+
+The total gas cost for sortiton increases linearly with group size by at least
+20k gas per member. Requiring more signers to sign messages makes it harder for
+adversaries to take over the group, but makes it so that the pool is less
+resilient to undelegation or operators going offline. Raising the
+strong-heartbeat threshold decreases the chance that you'll not have enough
+signers to sign transactions, but decreases a wallets lifespan and increases
+overhead. Lowering the dkg threshold increases the chance that we're able to
+create a wallet successfully but gives us a wallet with potentially fewer
+operators and a lower lifespan.
+
+Also important is optics - even though a group size of 70 with 40 signers
+required to sign messages might have all of the same security properties as a
```

Comment:
Not entirely; a 40/70 might be equally secure against being controlled by an adversary of around 25% power as 51/100, but it won't be as secure against becoming unavailable due to operators dropping out or the adversary trying to DoS it.

Reply:
Good point - I'll be more specific here

Reply:
Alrighty - rewrote this in b1b3c8d